docs/superpowers/plans/2026-09-04-genericize-a1-runtime.md

```
@@ -0,0 +1,557 @@
+3. Otherwise it exits non-zero, suggesting `PROSE_LINT_RUNTIME` or
+   `PROSE_LINT_BACKEND=url`.
+
+Check what it resolved to without starting anything:
+
+```sh
+bin/prose-lint-server.sh runtime   # prints e.g. podman
+```
+
+Rootless podman and nerdctl work: the container binds to `127.0.0.1` and the
+`--restart unless-stopped` policy is supported by all three runtimes.
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)